src/main/java/org/jitsi/jicofo/auth/AbstractAuthAuthority.java

```diff
@@ -83,8 +83,8 @@
     /**
      * The map of user JIDs to {@link AuthenticationSession}.
      */
-    private Map<String, AuthenticationSession> authenticationSessions
-            = new HashMap<String, AuthenticationSession>();
+    private final Map<String, AuthenticationSession> authenticationSessions
+            = new ConcurrentHashMap<>();
```

Comment:
Is this because of getSession(String)? Because authenticationSessions is explicitly synchronized on syncRoot otherwise.